test/distributed/fsdp/test_fsdp_state_dict.py

```diff
@@ -478,7 +493,10 @@ def test_state_dict_with_ignored_modules(self):
         # buffer still have the same data pointer
         with FSDP.state_dict_type(fsdp_model, StateDictType.FULL_STATE_DICT):
             sd2 = fsdp_model.state_dict()
-        for tensor, tensor_name in ignored_tensor_to_tensor_name.items():
+        for tensor, tensor_name in {
+            **ignored_tensor_to_tensor_name,
+            **buffer_to_buffer_name,
```

Comment:
I'm assuming that previously including buffers in this check would have raised an error?